archive/Finetuning-best-practices-guide.html

```
@@ -0,0 +1,2652 @@
+                <td>データ品質に強く依存、繰り返し学習のコスト</td>
+              </tr>
+            </tbody>
+          </table>
+          <p>
+            3つの手法は排他的ではなく、実務では「まずプロンプトエンジニアリングを試し、知識不足が原因ならRAGを追加し、それでも振る舞いが期待通りにならない場合にファインチューニングを検討する」という順序で組み合わせるのが定石です。
+          </p>
+
+          <div class="mermaid-container">
+            <pre class="mermaid-source">
+flowchart TD
+    A["LLMの出力を改善したい"] --> B{"モデルが知らない知識\n最新情報が必要か"}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)